docs/src/pages/components/timeline/timeline.md

```diff
@@ -43,9 +43,9 @@ The timeline can display content on opposite sides.
 
 {{"demo": "pages/components/timeline/OppositeContentTimeline.js"}}
 
-## Customized timeline
+## Complex timeline
```

Comment:
Can we stick with the previous name? What is simple or complex is subjective. It seems that "customized" fits the purpose of the demo pretty well. Could you elaborate why you changed it?

Reply:
>  I would also rename the demo to ## Complex timeline, it's still Material Design, default look.

The text still talks about customization and even links to it. I don't think we need a strong notion in "customization" that it looks different.

Reply:
Okay, I will change it back. I see that other components also use "Customization" in Lab demos.